### ISSUE TITLE
Upload/sketch multiple masks instead of single

### ISSUE THREAD

**Issue by space-nuko** [kohya-ss/sd-webui-additional-networks#144]
Seems like it would get around the limitation that you can only compose 3 LoRAs?

**Comment by kohya-ss** [comment]
I agree that. Doing so would take up a lot of space on the UI, which is annoying, but it is an easy and straightforward solution.
